index.js

```diff
@@ -0,0 +1,236 @@
+        cell.innerHTML = ""
+        cell.style.backgroundColor = "white"
+    }
+    //resetting arrays
+    boardArr = [0, 1, 2, 3, 4, 5, 6, 7, 8]
+    playerOneArr = []
+    playerTwoArr = []
+    winCom = []
+    //setting up clock
+    interval = setInterval(() => { updateClock() }, 1000)
+    clockCount = 0
+    clock.innerHTML = `Time Elapse: 0 seconds`
+    //two players
+    console.log(playerOneName)
+    playerOneDisplay = (playerOneName.value === "" ? "X" : playerOneName.value)
+    playerTwoDisplay = (playerTwoName.value === "" ? "O" : playerTwoName.value)
+    playerOneName.disabled = true
+    playerTwoName.disabled = true
+    showPlayer()
+}
+
+//adding event listener to start button
+startButton.addEventListener('click', start);
+
+//adding event listener to player/computer
+playerComputer.addEventListener('click', disPlayerPlayer)
+
+//funtion to disable player vs player box
+function disPlayerPlayer() {
+    playerPlayer.style.display ="none"
+    startButton.disabled = false
+    playerTwoName.style.display = "none"
+    gameMode = 'pvc' // set the gameMode to player vs computer
+    console.log(`THE GAMEMODE IS NOW: ${gameMode}`)
+}
+
+//adding event listener to player/player
+playerPlayer.addEventListener('click', disPlayerComputer)
+
+//funtion to disable player vs computer box
+function disPlayerComputer() {
+    playerComputer.style.display = "none"
+    startButton.disabled = false
+    gameMode = 'pvp' // Set the gameMode to player vs player
+    console.log(`THE GAMEMODE IS NOW: ${gameMode}`)
+}
+
+//function to update clock
+function updateClock() {
+    clockCount++
+    clock.innerHTML = `Time Elapse: ${clockCount} seconds`
+}
+
+//function to change status
+function showPlayer() {
+    if (currentPlayer === 1) {
+        gameStatus.innerHTML = `Player ${playerOneDisplay}'s turn`
+    } else {
+        gameStatus.innerHTML = `Player ${playerTwoDisplay}'s turn`
+    }
+}
+
+// function to prevent clicking on the same cell
+function alreadyClicked() {
+    gameStatus.innerHTML = `Please select an empty cell.`
+}
+
+//function to check for wins
+function checkWin(currentPlayerArr) {
+
+    let winner = false
+    wins.forEach((win) => {
+        if (currentPlayerArr.includes(win[0])) {
+            if (currentPlayerArr.includes(win[1])) {
+                if (currentPlayerArr.includes(win[2])) {
+                    winCom.push(...win)
+                    console.log(winCom)
+                    console.log("You Won!")
+                    winner = true
+                }
+            }
+        }
+    })
+    return winner
+}
+
+//prevents from clicking after win
+function reStart() {
+    for (let cell of cells) {
+        cell.removeEventListener('click', clicked)
+        cell.removeEventListener('click', alreadyClicked)
+    }
+    startButton.disabled = false
+    playerOneName.disabled = false
+    playerTwoName.disabled = false
+    playerOneName.value = ""
+    playerTwoName.value = ""
+}
+
```

Comment:
As part of your `reStart` function, be sure to re-enable the game-type options (i.e. return the `player vs player` and `player vs computer` options to their original display).  Otherwise, when you choose a game type, that's it! The option to choose a different game type each time would be great!
Also, I noticed that by clicking on the game type (`player vs player`) again after the game starts, it re-enables the start button, which causes the clock to speed up.  
Remove the event listener from the game type selector as well, and then re-enable it when the game is over.